src/RPCServer.ts

```diff
@@ -214,10 +209,10 @@ class RPCServer extends EventTarget {
     this.filterSensitive = filterSensitive || rpcUtils.filterSensitive;
   }
 
-  public async destroy(force: boolean = true): Promise<void> {
+  public async stop(force: boolean = true): Promise<void> {
```

Comment:
Can you change this to `{ force?: boolean }` as per @amydevs asked for already.

Reply:
Its force and reason right? as in `{force = true, reason}: {force?: boolean, reason?: any} = {}`